src/sentry/dynamic_sampling/rules_generator.py

```diff
@@ -20,6 +21,10 @@
 )
 from sentry.models import Project, Release
 
+# https://kubernetes.io/docs/reference/using-api/health-checks/
+# Also it covers: healthz, livez, readyz
+HEALTH_CHECK_GLOBS = ["*healthcheck*", "*healthy*", "*live*", "*ready*"]
```

Comment:
I'm not sure if these patterns are too generic, but I don't have better suggestions.

Reply:
Problem you can't find generic rules, cause each language or framework dictates it's own rules. 
E.g. you can find  `index.html` in AWS ELB or `/up` is some famous web frameworks. In V2 we plan to make it based on you transaction names or ask you to configure via UI.